js/src/base.js

```diff
@@ -12,10 +12,13 @@ for use within the Jupyter notebook interface.
 var widgets = require("@jupyter-widgets/base");
 var control = require("@jupyter-widgets/controls");
 var _ = require("underscore");
+var THREE = require("three");
 var three = require("./appthree");
+var thapp = require("./app");
```

Comment:
Should this be commented out? There is no other reference to thapp.

Reply:
Yeah, I am still experimenting with some modern JS syntax while testing new changes but I think backbone.js is sort of incompatible with newer JS syntax.